src/usr/local/emhttp/plugins/tailscale/include/Tailscale/System.php

```
@@ -66,6 +66,39 @@ public static function checkWebgui(Config $config, string $tailscale_ipv4): void
+        $httpsPort = -1;
+
+        if (($ident_config['USE_SSL'] ?? "no") != "no") {
+            $httpsPort = intval($ident_config['PORTSSL'] ?? 443);
+        }
+
+        $localAPI    = new LocalAPI();
+        $serveConfig = $localAPI->getServeConfig();
+
+        $tcpConfig = $serveConfig->TCP ?? array();
+
+        foreach ($tcpConfig as $key => $val) {
```

PHPStan (2.0.3)

Argument of an invalid type mixed supplied for foreach, only iterables are supported.


(foreach.nonIterable)